docs/gallery/interactive/pyfar_coordinates.ipynb

```diff
@@ -0,0 +1,395 @@
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Supported Coordinate systems\n",
+    "\n",
+    "Coordinate systems come in different flavors, like cartesian, spherical or cylindrical, where spherical has a couple of subdefinitions. The image below shows the available coordinate systems.\n",
+    "![alt text](../../resources/coordinate_systems.png)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Entering coordinate points\n",
+    "\n",
+    "You can input coordinate points manually into the system. By default, the system assumes Cartesian coordinates. Internally, all points are consistently stored in the Cartesian coordinate system."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "c = pf.Coordinates(np.arange(-5., 6), 0, 0)\n",
+    "# plot the sampling points\n",
+    "c.show()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To initialize `Coordinates` objects for different coordinate systems, you can use specific constructors like `pf.Coordinates.from_spherical_elevation(azimuth, elevation, radius)`. The naming convention for these constructors is always `pf.Coordinates.from_coordinate_system(red, green, blue)`, where `coordinate_system` should be replaced with the desired coordinate system, and `red`, `green`, `blue` describe the order of the coordinate properties. Remember, angles are always defined in radians.\n",
+    "\n",
+    "For more details, you can refer to the [coordinate class documentation](https://pyfar.readthedocs.io/en/latest/classes/pyfar.coordinates.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "azimuth_angles = np.arange(0, 2*np.pi, np.pi/20)\n",
+    "c1 = pf.Coordinates.from_spherical_elevation(azimuth_angles, 0, 1)\n",
+    "c1.show()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Meta data\n",
+    "\n",
+    "Lets have a look on the other data. At first we can print the coordinate object.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(c1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can observe that there are 40 points in total over all dimensions. This refers to the attribute `csize`.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "c1.csize"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The cshape of the data is (40,). The term cshape refers to the shape of a single coordinate."
```

Comment:
I think, in this way it would be more complex then nessesary. The coords are even saves in 3 separte variables `_x`, `_y`, and `z`. But i think the people must just understand that the cshape is the cshape of any of the coordinates. I added a bit more information to make it more clear.